### ISSUE TITLE
taupage-init: ERROR: Failed to start 10-prepare-disks.py

### ISSUE THREAD

**Issue by porrl** [zalando/planb-cassandra#20]
When starting nodes:
```
taupage-init: Deleting /home/ubuntu/.ssh/authorized_keys
taupage-init: INFO: Nothing to do for disk /dev/xvdf
[ 32.755692] EXT4-fs (xvdf): VFS: Can't find ext4 filesystem
[ 32.758308] EXT4-fs (xvdf): VFS: Can't find ext4 filesystem
[ 32.760762] EXT4-fs (xvdf): VFS: Can't find ext4 filesystem
[ 32.763306] FAT-fs (xvdf): invalid media value (0x2d)
taupage-init: mount: block device /dev/xvdf is write-protected, mounting read-only
[ 32.786130] EXT4-fs (xvdf): VFS: Can't find ext4 filesystem
[ 32.788717] EXT4-fs (xvdf): VFS: Can't find ext4 filesystem
[ 32.791114] EXT4-fs (xvdf): VFS: Can't find ext4 filesystem
[ 32.793472] FAT-fs (xvdf): invalid media value (0x2d)
taupage-init: mount: you must specify the filesystem type
taupage-init: Traceback (most recent call last):
taupage-init:   File "./init.d/10-prepare-disks.py", line 223, in <module>
taupage-init:     main()
taupage-init:   File "./init.d/10-prepare-disks.py", line 219, in main
taupage-init:     iterate_mounts(config)
taupage-init:   File "./init.d/10-prepare-disks.py", line 131, in iterate_mounts
taupage-init:     mount_partition(partition, mountpoint, options, filesystem, dir_exists(mountpoint), already_mounted)
taupage-init:   File "./init.d/10-prepare-disks.py", line 107, in mount_partition
taupage-init:     subprocess.check_call(mount_command)
taupage-init:   File "/usr/lib/python3.4/subprocess.py", line 557, in check_call
taupage-init:     raise CalledProcessError(retcode, cmd)
taupage-init: subprocess.CalledProcessError: Command '['mount', '-o', 'noatime,nodiratime', '/dev/xvdf', '/mounts/var/lib/cassandra']' returned non-zero exit status 32
```

**Comment by a1exsh** [comment]
Hm, what is the Taupage AMI version here?

**Comment by porrl** [comment]
Taupage-AMI-20160322-092814 (ami-e0860293)

**Comment by a1exsh** [comment]
Hm... It worked for me(TM) just fine with the same AMI version.

Did I forget to push a local commit?

**Comment by porrl** [comment]
Did you?

**Comment by a1exsh** [comment]
Unfortunately, I don't know. I'm on a sick leave till next week and my work
laptop is locked in the office. Sorry.

**Comment by a1exsh** [comment]
Ah, now I see.  Sorry, this EBS data volume feature was merged prematurely: I've expected that a fixed Taupage AMI will be already available, but the fix was reverted.  We need to prod STUPS guys to release a new Taupage version.

**Comment by MALPI** [comment]
Hey, is this already sorted out?

**Comment by a1exsh** [comment]
Not yet, ETA is this week.

**Comment by MALPI** [comment]
Any Update?

**Comment by a1exsh** [comment]
@fmueller are there any news on new Taupage AMI?

**Comment by fmueller** [comment]
Hey @a1exsh, I'm sure you're meaning @femueller from Stups team ;)

**Comment by femueller** [comment]
@a1exsh @MALPI AMI will be rolled out on Wednesday after one week of testing

**Comment by MALPI** [comment]
@femueller did it happen? :D

**Comment by a1exsh** [comment]
@MALPI this is now fixed with the release on a new Taupage-AMI-20160503-084000.  Kudos go to @femueller for making it happen! :-)
